qualtran/bloqs/data_loading/select_swap_qrom.py

```diff
@@ -451,15 +454,15 @@ def wire_symbol(self, reg: Optional[Register], idx: Tuple[int, ...] = tuple()) -
 def _qroam_multi_data() -> SelectSwapQROM:
     data1 = np.arange(5)
     data2 = np.arange(5) + 1
-    qroam_multi_data = SelectSwapQROM.build_from_data([data1, data2])
+    qroam_multi_data = SelectSwapQROM.build_from_data(data1, data2)
     return qroam_multi_data
 
 
 @bloq_example
 def _qroam_multi_dim() -> SelectSwapQROM:
     data1 = np.arange(25).reshape((5, 5))
     data2 = (np.arange(25) + 1).reshape((5, 5))
-    qroam_multi_dim = SelectSwapQROM.build_from_data([data1, data2])
+    qroam_multi_dim = SelectSwapQROM.build_from_data(data1, data2)
```

Comment:
Was this secretly failing?

Reply:
This was secretly passing multi-dimensional data instead of multiple datasets. My intention at the time of writing the test was to load 2 classical datasets; each of 2 dimensions. This was loading 1 classical dataset, of shape `(2, 5, 5)`.